server/src/main/java/com/coincoinche/store/InMemoryGameStore.java

```diff
@@ -0,0 +1,20 @@
+package com.coincoinche.store;
+
+import com.coincoinche.engine.CoincheGame;
+import java.util.HashMap;
+
+public class InMemoryGameStore implements GameStore {
+  private HashMap<String, CoincheGame> store;
+
+  public InMemoryGameStore() {
+    this.store = new HashMap<>();
+  }
+
+  public void saveGame(String gameId, CoincheGame game) {
+    this.store.put(gameId, game);
+  }
+
+  public CoincheGame getGame(String gameId) {
+    return this.store.get(gameId);
+  }
+}
```

Comment:
Is this thread-safe? If not, maybe we should add a `TODO` comment?

Reply:
Probably not, I created a ticket on Trello to have a multithreaded implementation for the server, rather than adding TODOs on all the class that would need it.